### ISSUE TITLE
Applied Energistics 2 Facade Compatability

### ISSUE THREAD

**Issue by mossb1** [mekanism/Mekanism-Feature-Requests#653]
### Describe the feature you'd like

A lot of times, Mekanism is included with AE2 and having the cables from Mekanism work with AE2 facades would be very nice for making builds look nicer. 

### Describe alternatives you've considered

If this could not be achieved, a similar type facade would be nice.

**Comment by chuman72486** [comment]
I agree.  Any mod with cables/pipes should have an option for facades.  AE2 and many others have their own mod specific facades, and there's a couple third party mods that claim to add facades to Mekanism, but I think it should be something built into the mod itself.  Having a native facade on item pipes can also be useful so that the item entities in the pipes don't need to be rendered for covered pipes.  Configurator in Wrench mode can be used to remove facades without breaking the pipes.
